third_party/gRPC_utils/cli_call.cc

```diff
@@ -27,6 +27,7 @@
 #include <optional>
 // END MODIFIED
 
```

Comment:
I took the most recent `cli_call.cc` from gRPC example source code and adapted it again for our needs.